runtime/doc/vim_diff.txt

```diff
@@ -39,6 +39,7 @@ the differences.
 - 'display' defaults to "lastline,msgsep"
 - 'encoding' is UTF-8 (cf. 'fileencoding' for file-content encoding)
 - 'fillchars' defaults (in effect) to "vert:│,fold:·,sep:│"
+- 'foldcolumn' defaults to "auto"
```

Comment:
Is this true? Looks like it is "0" in options.lua. https://github.com/neovim/neovim/pull/11716/files#diff-2b89728238835a06b3522ac53e43510bR838

And it should probably stay "0", we don't have a strong reason to change it.